tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs

```diff
@@ -3742,11 +3744,6 @@ public void AiMonitoringStreamingDisabledByEnvironmentVariableWhenAiMonitoringEn
             Assert.That(_defaultConfig.AiMonitoringStreamingEnabled, Is.False);
         }
 
-        [Test]
```

Comment:
Isn't this still a valid test? Though now that I look at the spec again, isn't it supposed to be enabled by default?

Reply:
This test was essentially a duplicate of `AiMonitoringRecordContentDisabledWhenAiMonitoringDisabled`